ReactiveXComponent/ReactiveXComponent.csproj

```diff
@@ -1,169 +1,36 @@
-    <Compile Include="RabbitMq\RabbitMqPublisher.cs" />
-    <Compile Include="IXComponentApi.cs" />
-    <Compile Include="Parser\XCApiConfigParser.cs" />
-    <Compile Include="Parser\TopicIdentifier.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="RabbitMq\RabbitMqSession.cs" />
-    <Compile Include="RabbitMq\RabbitMqSubscriber.cs" />
-    <Compile Include="Common\ReactiveXComponentException.cs" />
-    <Compile Include="Serializer\BinarySerializer.cs" />
-    <Compile Include="Serializer\BsonSerializer.cs" />
-    <Compile Include="Serializer\CustomSerializerAttribute.cs" />
-    <Compile Include="Serializer\CustomSerializerContainerAttribute.cs" />
-    <Compile Include="Serializer\ISerializer.cs" />
-    <Compile Include="Serializer\JsonSerializer.cs" />
-    <Compile Include="Serializer\SerializationType.cs" />
-    <Compile Include="Serializer\SerializerFactory.cs" />
-    <Compile Include="Serializer\XCSerializationException.cs" />
-    <Compile Include="WebSocket\IWebSocketClient.cs" />
-    <Compile Include="WebSocket\WebSocketClient.cs" />
-    <Compile Include="WebSocket\WebSocketCommand.cs" />
-    <Compile Include="WebSocket\WebSocketConnection.cs" />
-    <Compile Include="WebSocket\WebSocketGetXcApiResponse.cs" />
-    <Compile Include="WebSocket\WebSocketMessage.cs" />
-    <Compile Include="WebSocket\WebSocketMessageHelper.cs" />
-    <Compile Include="WebSocket\WebSocketPacket.cs" />
-    <Compile Include="WebSocket\WebSocketPublisher.cs" />
-    <Compile Include="WebSocket\WebSocketSession.cs" />
-    <Compile Include="WebSocket\WebSocketSnapshotManager.cs" />
-    <Compile Include="WebSocket\WebSocketSnapshotMessage.cs" />
-    <Compile Include="WebSocket\WebSocketSubscriber.cs" />
-    <Compile Include="WebSocket\WebSocketSubscription.cs" />
-    <Compile Include="WebSocket\WebSocketTopic.cs" />
-    <Compile Include="WebSocket\WebSocketTopicKind.cs" />
-    <Compile Include="WebSocket\WebSocketXCApiManager.cs" />
-    <Compile Include="WebSocket\WebSocketGetXcApiListResponse.cs" />
-    <Compile Include="WebSocket\WebSocketXCApiRequest.cs" />
-    <Compile Include="WebSocket\WebsocketXCApiCommand.cs" />
-    <Compile Include="XComponentApi.cs" />
+    <PackageReference Include="ILRepack.MSBuild.Task" Version="2.0.13" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
+    <PackageReference Include="RabbitMQ.Client" Version="5.1.0" />
+    <PackageReference Include="SuperSocket.ClientEngine" Version="0.10.0" />
+    <PackageReference Include="System.Reactive.Linq" Version="4.1.2" />
+    <PackageReference Include="WebSocket4Net" Version="0.15.2" />
   </ItemGroup>
-  <ItemGroup>
-    <None Include="app.config" />
-    <None Include="packages.config" />
-  </ItemGroup>
-  <ItemGroup />
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(SolutionDir)\BuildCommon.targets" Condition="'$(SetAssemblyVersion)'=='true'" />

```

Comment:
The assemblies versions are no longer set if we remove this part. Do we have a new way of setting assembly version?

Reply:
Update with last commit